.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
```

Comment:
El nombre debería ser algo como: Build, lint and test
(Se realizan estas 3 tareas).